install.js

```diff
@@ -4,8 +4,17 @@ var chalk = require('chalk')
 
 var options = require('./index.js').getOptions()
 
-if (options && options.nohook) {
-  console.log('commitplease: package.json or .npmrc set to skip hook')
+// Is this a self-install? If so, do not copy hooks and quit early
+var pkgPath = path.join(options.projectPath, 'package.json')
+var pkg = fs.existsSync(pkgPath) && require(pkgPath)
+
+if (pkg && pkg.name && pkg.name === 'commitplease') {
+  console.log('commitplease: self-install detected, skipping hooks')
```

Comment:
I was surprised that this was missing, apparently we lost it earlier - I remember writing a similar check originally. After a bit of digging I found the original code: https://github.com/jzaefferer/commitplease/commit/2021ce33ab4e3fee6384e678090cd07fcbe15859#diff-fcd5628e27bdebc81efd345a905292c3L1

I guess this new implementation using `options.projectPath` makes more sense, now that we have that in place. I didn't notice anything from the old implementation that would change this one - do you?